style.css

```diff
@@ -332,10 +332,17 @@ footer {
 
 #toc {
 	position: fixed;
-	left: 1%;
-	max-width: calc(48% - 450px);
+	left: 0;
+	bottom: 15px;
+	max-width: calc(48% - 450px - 30px);
```

Comment:
Now that `left` is 0, we can change this to `calc(50% - 450px - 40px)`.

`50%` because `48%` was used to have a distance of `1%` left and right.

And `40px` because I made a mistake. The greed checks are 40px wide.

Reply:
Ahh, that explains why it did overlap the "§" at some size.